DemoApplication/FormMain.cs

```diff
@@ -125,7 +126,7 @@ private void btnDeserializeFromFile_Click(object sender, EventArgs e)
             OnDeserialize(true);
         }
 
-        private void OnDeserialize(bool openFromFile)
+         private void OnDeserialize(bool openFromFile)
```

Comment:
nitpick, strange space?

Reply:
At last I found the blank I lost today, lol
Will fix